hw/drivers/sensors/lis2dw12/src/lis2dw12.c

```diff
@@ -2047,8 +2042,6 @@ lis2dw12_config(struct lis2dw12 *lis2dw12, struct lis2dw12_cfg *cfg)
         goto err;
     }
     lis2dw12->cfg.tap_cfg = cfg->tap_cfg;
-
-    lis2dw12_write8(itf, 0x34, 0x80);
```

Comment:
Is this necessary inside of tap configuration, or just not necessary at all?

Reply:
I don't believe it is necessary at all, single and double tap detection work fine without it.